packages/rpc-core/src/rpc-methods/getHighestSnapshotSlot.ts

```diff
@@ -0,0 +1,20 @@
+import { Slot } from './common';
+
+type GetHighestSnapshotSlotApiResponse = Readonly<{
+    full: Slot;
+    incremental: Slot | null;
+}>;
```

Comment:
Looks like this is correct, but it might be worth PRing a fix to the docs which have it as `incremental: <u64|undefined>`

Reply:
I actually meant to do that shortly!